staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/etcd.go

```diff
@@ -50,5 +88,154 @@ func NewREST(resource schema.GroupResource, listKind schema.GroupVersionKind, st
 	if err := store.CompleteWithOptions(options); err != nil {
 		panic(err) // TODO: Propagate error up
 	}
-	return &REST{store}
+
```

Comment:
presence of status should affect create/update strategy on the main resource to disallow changes to status stanza, right?

Reply:
Create - https://github.com/kubernetes/kubernetes/pull/55168/files#diff-34aedeefa7784abbe7531bae68833ba6R74

Update - https://github.com/kubernetes/kubernetes/pull/55168/files#diff-34aedeefa7784abbe7531bae68833ba6R96